docs/reference-guide/backup.md

```diff
@@ -4,69 +4,88 @@ title: Data durability and backup
-of older files. 
+Memgraph has WAL enabled by default. To switch it on and off, use the boolean
+`storage-wal-enabled` flag. Other WAL-related flags are
+`storage-wal-file-flush-every-n-tx` and `storage-wal-file-size-kib`; their uses
+are described [here](/docs/memgraph/how-to-guides/config-logs).
 
-Behavior of these mechanisms can be tweaked in the configuration file
-(`/etc/memgraph/memgraph.conf`) by changing values of various flags regarding
-storage.
+### Snapshots
 
-Check the configuration reference guide for the [possible storage configuration
-settings](../reference-guide/configuration#storage) and if you need
-help changing the configuration, check out the how-to guide on [configuration
-management](/how-to-guides/config-logs.md). 
+Snapshots provide a faster way to restore the states of your database. Memgraph
+periodically takes snapshots during runtime. When a snapshot is triggered, the
+entire data storage is written to the drive.
+On startup, the database state is recovered from the most recent snapshot file.
+The timestamp of the snapshot is compared with the latest update recorded in
+the WAL file and, if the snapshot is less recent, the state of the DB will be
+recovered using the WAL file.
+
+Memgraph has snapshot creation enabled by default. You can configure the exact
+snapshot creation behavior by defining the relevant flags
+([reference](/docs/memgraph/reference-guide/configuration#storage)).
+Alternatively, you can make one directly by running the following query:
+
+```opencypher
+CREATE SNAPSHOT;
+```
 
 :::caution
-Snapshot and WAL files are (currently) not compatible between Memgraph
-versions.
+Snapshots and WAL files are presently not compatible between Memgraph versions.
 :::
 
+### Data directory
+
+The data directory is the location where Memgraph saves write-ahead logs and
+snapshots; it functions as the dedicated site for permanent data.
+
+The default data directory path is `var/lib/memgraph`. You can change the path
+by setting the `data-dir` configuration flag
+([reference](/docs/memgraph/reference-guide/configuration#other)).
+
 ## Backup
 
-Snapshots of the Memgraph instance are created periodically based on the time
-defined with the `--storage-snapshot-interval-sec` configuration flag. If you
-want to generate a snapshot for the current database state instantly, use the
-following query:
+You can easily back up Memgraph by following a three-step process:
 
-```opencypher
-CREATE SNAPSHOT;
-```
+1. Create a snapshot
+2. Lock the data directory
+3. Copy the snapshot to the backup location and unlock the directory
 
-To create a backup of the instance, disable deletion of durability files, then
-copy the **data directory** or individual WAL or snapshot files to a backup
-location. 
+A detailed guide is available
+[here](/docs/memgraph/how-to-guides/create-backup).
 
-To disable deletion, run the following query:
-```opencypher
-LOCK DATA DIRECTORY;
-```
+## Database dump
```

Comment:
The reference guide should always contain the most information related to the subject... so I think I would leave the queries instructing how to lock and unlock the data directory? How to guide shouldn't have any information that is not available in the reference guide

Reply:
Makes sense, added that back